runtime/src/lib.rs

```diff
@@ -320,6 +320,24 @@ impl pallet_sudo::Config for Runtime {
 	type RuntimeCall = RuntimeCall;
 }
 
+parameter_types! {
+	// One storage item; key size is 32; value is size 4+4+16+32 bytes = 56 bytes.
+	pub const DepositBase: Balance = (1) as Balance * 2_000 * 10_000_000 + (88 as Balance) * 100 * 1_000_000;
+	// Additional storage item size of 32 bytes.
+	pub const DepositFactor: Balance = (0) as Balance * 2_000 * 10_000_000 + (32 as Balance) * 100 * 1_000_000;
```

Comment:
Do we know what these two do? Guessing `DespositBase` is the base amount of currency needed to reserve to create a multisig?

Reply:
Yes,  this is the implementation of the deposit function from polkadot's runtime. The deposit function has two arguments, items and bytes. It's an ease-of-use function which is used to calculate per-byte fees when storage data is used on chain.